### PR TITLE
Fixed solhint config for Travis-CI and fixed contract codes according to solhint rules

### PR DESCRIPTION
**Pull request by ayobuenavista**

### CODE REVIEW

File: contracts/sol6/wrappers/KyberRateHelper.sol [hunk 1]
```diff
@@ -7,7 +7,6 @@ import "../IKyberStorage.sol";
 import "../utils/Utils5.sol";
 import "../utils/WithdrawableNoModifiers.sol";
 
-
```
Comment (ilanDoron) [inline contracts/sol6/wrappers/KyberRateHelper.sol:10]: why one line?

File: contracts/sol6/wrappers/KyberRateHelper.sol [hunk 2]
```diff
@@ -172,11 +163,11 @@ contract KyberRateHelper is IKyberRateHelper, WithdrawableNoModifiers, Utils5 {
         uint256 optionalSellAmount,
         uint256 networkFeeBps
     ) internal view returns (bytes32[] memory sellReserves, uint256[] memory sellRates) {
-        Amounts memory A;
+        Amounts memory amts;
```
Comment (ilanDoron) [inline contracts/sol6/wrappers/KyberRateHelper.sol:166]: might as well call it amounts
amts is non conventional.
as bad as A

Comment (ayobuenavista) [reply contracts/sol6/wrappers/KyberRateHelper.sol]: Fixed

File: contracts/sol6/wrappers/KyberRateHelper.sol (hunk 1 above)
Comment (ayobuenavista) [reply contracts/sol6/wrappers/KyberRateHelper.sol]: Overlooked this. Fixed.

**Comment by ilanDoron** [comment]
Unfortunately we can't merge this work to Katalyst.

please keep solhint config file

We will later use it for Katalyst mock contracts and for next project.

**Pull request closed without merging**